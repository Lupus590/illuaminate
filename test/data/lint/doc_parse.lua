-- config: (lint (only doc:malformed-tag doc:malformed-type doc:unknown-flag doc:unknown-tag doc:duplicate-definitions doc:bad-index doc:wrong-tag))

--- A module
--
-- @module foo
-- @module bar
-- @module[x]

--- @unknown_standalone
