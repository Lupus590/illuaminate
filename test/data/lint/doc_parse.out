doc_parse.lua:[3:0-7:13]: Duplicate @module definitions (named 'foo' and '') [doc:duplicate-definitions]
   │
 3 │ --- A module
   │ ^^^^^^^^^^^^
doc_parse.lua:[3:0-7:13]: @module has unknown flag 'x' [doc:unknown-flag]
   │
 3 │ --- A module
   │ ^^^^^^^^^^^^
doc_parse.lua:[3:0-7:13]: Duplicate @module definitions (named 'foo' and 'bar') [doc:duplicate-definitions]
   │
 3 │ --- A module
   │ ^^^^^^^^^^^^
3 errors and 0 warnings
